certification/internal/k8s/deployable_by_olm_mounted.go

```diff
@@ -75,7 +75,7 @@ func (p *DeployableByOlmMountedCheck) setUp(bundle string) error {
 	}
 
 	// retrieve the operator metadata from bundle image
-	data, err := containerutil.GetAnnotationsFromBundle(bundle)
```

Comment:
I wouldn't bother with these changes, as this code is going to go away very soon. I'd rather not see it in the same commit.

Reply:
sure, I can revert this change.